From f70c7bccd8551da007dd7c3722977ec0ac8f7a55 Mon Sep 17 00:00:00 2001
From: Delphine CC Chiu <Delphine_CC_Chiu@wiwynn.com>
Date: Wed, 22 May 2024 09:28:50 +0800
Subject: [PATCH 1/2] schemas: Add CheckHysterWithSetpt for PID

Adding CheckHysterWithSetpt, an optional boolean parameter, which
has already been implemented and merged:
https://gerrit.openbmc.org/c/openbmc/phosphor-pid-control/+/58009

Change-Id: Iee6a12c9f3510fb9ba4a886c071cd85a358156e5
Signed-off-by: Delphine CC Chiu <Delphine_CC_Chiu@wiwynn.com>
---
 schemas/pid.json | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/schemas/pid.json b/schemas/pid.json
index 2db98c7..b64deec 100644
--- a/schemas/pid.json
+++ b/schemas/pid.json
@@ -4,6 +4,9 @@
         "Pid": {
             "type": "object",
             "properties": {
+                "CheckHysterWithSetpt": {
+                    "type": "boolean"
+                },
                 "Class": {
                     "type": "string"
                 },
-- 
2.25.1

